changes/tasks.py

```diff
@@ -1468,6 +1468,82 @@ def run(self, vumi_message_id: str, event_type: str, **kwargs) -> None:
 process_whatsapp_system_event = ProcessWhatsAppSystemEvent()
 
 
+class ProcessWhatsAppTimeoutSystemEvent(Task):
+    """
+    Notifies a user that a message we send them on Whatsapp has not been delivered
+    within 7 days, she will be sent an outbound
+    message informing them and give the option to switch to SMS.
+    """
+
+    name = "ndoh_hub.changes.tasks.process_whatsapp_timeout_system_event"
+
+    def return_date_time(self):
+        return datetime.utcnow()
+
+    def send_outbound(self, user_id, source_id, identity_uuid):
+        identity = is_client.get_identity(identity_uuid)
+        # Transform to django language code
+        for item in identity["results"]:
```

Comment:
Since we're using `get_identity`, this won't have a `results` key with a list of identities. This will instead just be that single identity that we're looking for

Reply:
ahh yes thank you